js/global.js

```diff
@@ -1,27 +1,41 @@
 $(document).ready(
     function () {
 
+        // Note from Nathan 9/7/2022: I'm not sure why this exists? Legacy?
         let pageUrl = document.URL.split('/')[3];
         if (pageUrl === 'DSFA') {
             pageUrl = '/DSFA/'
         } else {
             pageUrl = '/';
         }
 
-        //  NAV LINKS
-        let apndNavItems = $('.navbar-items').find('a').first();
-
-        apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'profiles').html('Students');
-        apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'projects').html('Projects');
-        apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'events').html('Events');
-        apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'visits').html('Visits');
-        apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'about').html('About');
-        apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'contact').html('<i class=\"fas fa-envelope\"></i>');
-
-        $('.navbar-items').find('a').first().remove();
-        // END --  NAV LINKS
-
-
+        // INSERT NAV HTML
+        let headerElem = document.querySelector('#js-populate-header');
 
+        headerElem.innerHTML = `
+            <nav class="navbar">
+                <div class="navbar-logo">
+                    <a href="http://computingforall.org" class="cfa-logo"></a>
+                    <a href="http://dsfastudents.org/">
```

Comment:
This is a place where we ought to use a relative path if possible

Reply:
AH! Gotchya. I updated the code since it seemed relevant to the work.